core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1831,6 +1831,16 @@
 
   public static final short LOCAL_DICT_ENCODED_BYTEARRAY_SIZE = 3;
 
+  /**
+   * property to be used for specifying the max character limit for string/varchar data type till
```

Comment:
’specifying the max character limit‘ ->'specifying the max byte limit'

Reply:
ok